### PR TITLE
Docs for bessemer live area

### PR DESCRIPTION
**Pull request by willfurnass**
Rather light on detail and definitely need reviewing.

### CODE REVIEW

File: bessemer/software/development/gcc.rst [hunk 1]
```diff
@@ -1,13 +1,21 @@
...
-It is possible to switch versions of the gcc compiler suite using modules. After connecting to Bessemer,  start an interactive sesssion with the :code:``srun --pty bash –i`` command. Choose the version of the compiler you wish to use using one of the following commands ::
+The GNU Compiler Collection (gcc) is a widely used, free collection of compilers
+for C (gcc), C++ (g++) and Fortran (gfortran).
 
-    module use /usr/local/modulefiles/staging/eb/all/
-    module load GCC/8.2.0-2.31.1
+It is possible to switch versions of the gcc compiler suite using modules.
+After connecting to Bessemer,  start an interactive sessson: :: 
+
+   srun --pty bash –i`` command.
```
Comment (anthonybrookfield) [inline bessemer/software/development/gcc.rst]:
```suggestion
   srun --pty bash –i
```

File: bessemer/software/development/gcc.rst [hunk 2]
```diff
@@ -1,13 +1,21 @@
...
+It is possible to switch versions of the gcc compiler suite using modules.
+After connecting to Bessemer,  start an interactive sessson: :: 
+
+   srun --pty bash –i`` command. 
+
+then choose the version of the compiler you wish to use
+by running *one* of the following lines: ::
+
+   module load GCC/8.2.0-2.31.1
+   module load 7.3.0-2.30
```
Comment (anthonybrookfield) [inline bessemer/software/development/gcc.rst]:
```suggestion
   module load GCC/7.3.0-2.30  
```

File: bessemer/software/development/gcc.rst [hunk 3]
```diff
@@ -19,8 +27,10 @@ Language support
...
-man pages are available on the system. Once you have loaded the required version of `gcc`, type ::
+
+man pages are available on the system.
+Once you have loaded the required version of `gcc`, type ::
 
     man gcc
 
-* `What's new in the gcc version 8 series? <https://gcc.gnu.org/gcc-8/changes.html>`
\ No newline at end of file
+* `What's new in the gcc version 8 series? <https://gcc.gnu.org/gcc-8/changes.html>`
```
Comment (anthonybrookfield) [inline bessemer/software/development/gcc.rst]:
```suggestion
* `What's new in the gcc version 8 series? <https://gcc.gnu.org/gcc-8/changes.html>`__
```

File: bessemer/software/libs/scalapack.rst [hunk 4]
```diff
@@ -0,0 +1,34 @@
+.. _scalapack_bessemer:
+
+OpenBLAS
```
Comment (anthonybrookfield) [inline bessemer/software/libs/scalapack.rst]:
```suggestion
ScaLAPACK
```

File: bessemer/software/parallel/impi.rst [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+Write a shell script (minimal example) We name the script as ‘test.sh’: ::
+
+
+    #!/bin/bash
+    #SBATCH --nodes=1
+    #SBATCH --ntasks-per-node=40
+
+    module load OpenMPI/3.1.3-GCC-8.2.0-2.31.1
+
+    srun file
```
Comment (anthonybrookfield) [inline bessemer/software/parallel/impi.rst]:
```suggestion
    srun --export=ALL file
```

File: bessemer/software/parallel/openmpi.rst [hunk 6]
```diff
@@ -1,7 +1,9 @@
...
-=====================
+.. _openmpi_bessemer:
 
-.. sidebar:: OpenMPI (gcc version)
+OpenMPI
+=======
+
+.. sidebar:: OpenMPI
 
    :Latest Version: 3.1.1
```
Comment (anthonybrookfield) [inline bessemer/software/parallel/openmpi.rst]:
```suggestion
   :Latest Version: 3.1.3
```

File: bessemer/software/parallel/openmpi.rst [hunk 7]
```diff
@@ -97,7 +97,6 @@ Write a shell script (minimal example) We name the script as ‘test.sh’: ::
     #SBATCH --nodes=1
     #SBATCH --ntasks-per-node=40
 
-    module use /usr/local/modulefiles/staging/eb/all/
     module load OpenMPI/3.1.3-GCC-8.2.0-2.31.1
 
     srun file
```
Comment (anthonybrookfield) [inline bessemer/software/parallel/openmpi.rst]:
```suggestion
    srun --export=ALL file
```

File: bessemer/slurm.rst [hunk 8]
```diff
@@ -73,7 +73,6 @@ The next example script requests 40 CPU cores in total and 64Gb memory. Notifica
     #SBATCH --mem=64000
     #SBATCH --mail-user=username@sheffield.ac.uk
 
-    module use /usr/local/modulefiles/staging/eb/all/
     module load OpenMPI/3.1.3-GCC-8.2.0-2.31.1
 
     srun program
```
Comment (anthonybrookfield) [inline bessemer/slurm.rst]:
```suggestion
    srun --export=ALL program
```

**Comment by willfurnass** [comment]
Ta @anthonybrookfield.

**Pull request merged**